ocs_ci/helpers/storage_auto_scaler.py

```diff
@@ -0,0 +1,92 @@
+import logging
+
+from ocs_ci.ocs.exceptions import ResourceWrongStatusException
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.framework import config
+
+logger = logging.getLogger(__name__)
```

Comment:
plz consider add some logging messages when needed, currently "logger" is never used in the file